db/db-schema.test.sql

```diff
@@ -132,7 +132,7 @@ CREATE TABLE IF NOT EXISTS `test`.`REGISTRATION` (
   `veteran` VARCHAR(15) NULL DEFAULT 'false',
   `pin` INT(11) NOT NULL AUTO_INCREMENT,
   `time` VARCHAR(45) NULL DEFAULT NULL,
-  `hackathon` VARCHAR(45) NOT NULL DEFAULT '84ed52ff52f84591aabe151666fae240',
+  `hackathon` VARCHAR(45) NOT NULL DEFAULT '0caffc937e1c4501bcb9d8d7db06ccbd',
```

Comment:
Why did this need a change? And will it keep needing change in the future as the default hackathon changes?

Reply:
To be honest, I'm not sure what if this file is actually used or not, I've just been using it as a reference for foreign key dependencies. I think it's auto generated using mysql workbench anyway so I don't think this needed to be changed.

Reply:
I would recommend debugging the SQL a bit to see what's happening here.

Reply:
For documentation purposes, I checked and so far, the sql script is only used in a shell script to generate a test database and isn't used anywhere else in the current project or the Typescript migration pull request.